### PR TITLE
Update critical easymon checks

### PR DESCRIPTION
**Pull request by eric-pigeon**
Only elasticsearch, mongodb and redis are critical services for running
the application.

### CODE REVIEW

File: core/test/integration/workarea/monitoring_integration_test.rb [hunk 1]
```diff
@@ -3,27 +3,32 @@
 module Workarea
   class MonitoringIntegrationTest < Workarea::IntegrationTest
     def test_monitors_the_mongodb_status
-      get workarea.easymon_path('mongodb')
+      get workarea.easymon_path + '/mongodb'
```
Comment (bencrouse) [inline core/test/integration/workarea/monitoring_integration_test.rb]: Good catch.

**Comment by eric-pigeon** [comment]
Yes a patch would be better, can you update where this PR points to?

**Comment by mttdffy** [comment]
> Yes a patch would be better, can you update where this PR points to?

You'll have to rebase the commit

**Comment by bencrouse** [comment]
I cherry-picked this into `v3.4-stable`, closing this.

**Pull request closed without merging**